### PR TITLE
Chti update

### PR DESCRIPTION
**Pull request by Masteroshi430**

### CODE REVIEW

File: Labs/ItemStatBreakdownPlugin.cs [hunk 1]
```diff
@@ -229,6 +231,8 @@ public override void Load(IController hud)
             "On_Hit_Knockback_Proc_Chance",
             "On_Hit_Immobilize_Proc_Chance",
             "---------",
+	    "Damage_Percent_Bonus_Vs_Monster_Type  / 2"
```
Comment (djtms) [inline Labs/ItemStatBreakdownPlugin.cs]: add on end of line ","

**Review by djtms — COMMENTED** [review]
correct little exception :)

**Comment by Masteroshi430** [comment]
Dziękuję!

**Comment by JackCeparou** [comment]
The `  / 2` looks strange to me ;p

What's the monster type associated with this attribute code ? (beast, human, ect ?)

**Comment by Masteroshi430** [comment]
I don't remember if it's / 2 or \ 2 but 2 is beasts 
% Damage to beasts

**Comment by JackCeparou** [comment]
I don't forget this one, but I want to check before merging it ; )

**Pull request closed without merging**